### DISCUSSION TITLE
Load Transaction Log File

### DISCUSSION THREAD

**Discussion by macsir** [cubewise-code/tm1py#863] [Ideas]
Hi, @MariusWirtz  Do we have a plan to create a function to load standard transaction log file? I only see the functions to extract transaction log out. Do you have a better idea to restore/repeat based on transaction log via REST API? Thanks.

**Comment by MariusWirtz** [comment]
Hi @macsir,

I agree that would be a useful function.
Please feel free to request this feature in the issues tab as an enhancement.

Marius
